packages/components/src/AboutDialog/AboutDialogTable.component.js

```diff
@@ -0,0 +1,98 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import { Skeleton } from '../';
+import theme from './AboutDialog.scss';
+
+const maxHeight = 300;
+const modalWidth = 586;
+const serviceHeight = 30;
+
+export function Text({ text, loading, size = Skeleton.SIZES.medium }) {
+	return <div>{loading ? <Skeleton type={Skeleton.TYPES.text} size={size} /> : text}</div>;
+}
+
+/**
+ * This function set the column width that rely on the number of column displayed
+ * @param {array} definition
+ */
+function getColumnWidth(definition) {
+	return { minWidth: modalWidth / definition.length };
+}
+
+/**
+ * This function set the max height of the table in order to not have a blank space when
+ * the list of services is short
+ * @param {array} services
+ */
+function getTbodyStyle(services) {
+	return {
+		height: Math.min(maxHeight, services.length * serviceHeight),
+	};
+}
+
+export function AboutDialogTable({
+	t,
+	definition = [
```

Comment:
It deserves to use a composition pattern, isn't it?

Reply:
I've tried to use composition on this dialog but i haven't find a elegant solution to make it work as the composition has to be done for the header and the body, there is a lot of elements to put. Not sure it works well, but do you have something in mind ? :)